src/js/menu/menu-item.js

```diff
@@ -68,6 +70,24 @@ class MenuItem extends ClickableComponent {
     }, props), attrs);
   }
 
+  /**
+   * Ignore keys which are used by the menu, but pass any other ones up. See
+   * {@link ClickableComponent#handleKeyPress} for instances where this is called.
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that caused this function to be called.
+   *
+   * @listens keydown
+   */
+  handleKeyPress(event) {
+    if (!MenuKeys.some((key) => {
```

Comment:
You should be able to do this all on one line a bit cleaner with something like this: 
```suggestion
    if (!MenuKeys.some((key) => keycode.isEventKey(event, key)) {
```